packages/framework/src/shared/models.ts

```diff
@@ -129,6 +130,7 @@ export interface EnsembleThemeModel {
     } & { [key: string]: string };
   };
   Styles?: { [key: string]: CSSProperties };
+  [key: string]: unknown;
```

Comment:
Please fix this typing to reflect the proper structure.

Reply:
Hey, we need this. I added comment there to explain the usecase of this,
actually in theme file, there will be multiple themes like, Dark, Primary, Light and so on, so this is for that usecase

Reply:
Yes I understand, but this can be greatly improved since you know the structure of the theme - create the rest of the type not just make it `unknown`